pom.xml

```diff
@@ -167,6 +168,7 @@
 
     <build>
         <plugins>
+<!--
             <plugin>
                 <groupId>org.sonatype.plugins</groupId>
```

Comment:
I had to remove it because the Travis builds failed. Feel free to merge [the original commit](https://github.com/peterjanes/jsdoc3-maven-plugin/commit/5aecb93ff9c76a693cf0213231836b8236dcc985) once you've got the Travis config fixed.